cg/cli/workflow/microsalt/base.py

```diff
@@ -170,7 +170,7 @@ def run(
         analysis_api.set_statusdb_action(case_id=case_id, action="running")
         analysis_api.process.run_command(parameters=analyse_command, dry_run=dry_run)
     except:
-        LOG.error("Failed to run analysis!")
+        LOG.error(f"Failed to run analysis!")
```

Comment:
```suggestion
        LOG.error("Failed to run analysis!")
```
This does not have a variable in it.